content/features/features.rst

```diff
@@ -239,7 +65,7 @@ here <https://play.google.com/books/reader?id=dnzVBAAAQBAJ&printsec=frontcover&o
 if it is on 6.x, `click
 here <https://play.google.com/books/reader?id=tu6bCgAAQBAJ&printsec=frontcover&output=reader&hl=en&pg=GBS.PP1>`__.
 
-12.3.1. Use a Screen Lock
+12.2.1. Use a Screen Lock
```

Comment:
Can we do away with these numbers as part of this issue? Or were you going to do it when we change outlines?

Reply:
yes this will be done when moving to new outline